components/brave_ads/browser/notification_helper.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/notification_helper.h"
+
+namespace brave_ads {
+
+NotificationHelper::NotificationHelper() {}
+
+NotificationHelper::~NotificationHelper() {}
+
+bool NotificationHelper::IsNotificationsAvailable() const {
+  return true;
+}
+
+bool NotificationHelper::IsBraveAdsChannelEnabled() const {
```

Comment:
This is platform specific so should not be a part of this class. The helpers are in place to allow cross-platform implementations